crates/configuration/src/version5/native_operations.rs

```diff
@@ -225,40 +225,35 @@ pub async fn oids_to_typenames(
 
     // Reverse lookup the schema.typename and find the ndc type name,
     // if we find all we can just add the nq and call it a day.
-    for row in rows {
+    'rows: for row in rows {
         let schema_name: String = row.schema_name;
         let type_name: String = row.type_name;
         let oid: i64 = row.oid;
 
-        let mut found = false;
         for (scalar_type_name, info) in &configuration.metadata.types.scalar.0 {
             if info.schema_name == schema_name && info.type_name == type_name {
                 oids_map.insert(oid, scalar_type_name.inner().clone());
-                found = true;
-                continue;
```

Comment:
This continue statement did nothing.

There's a couple options for intent:
- either this was meant to be a `break`, to exit the inner loop on first match
- or, the intent was to proceed to the next iteration of the outer loop

I think the intent was the latter, so
- added label to loop
- continue using label so the right loop is continued
- removed `found` since that becomes redundant

We could, alternatively, use break, and keep `found` if we think that's more readable?

Reply:
Note that using break would mean we'd iterate over composite types always, even if we did find our type as a scalar

Using `continue 'rows` I think better matches the intent here:
- iterate over rows
- for each row, try to find a matching scalar type and insert that to the map
- if no scalar found, try to find a matching composite
- if neither found, use type name as is